coalib/bearlib/aspects/decorators.py

```diff
@@ -4,7 +4,9 @@
 from .taste import Taste
 from .meta import aspectclass
 from coalib.settings.FunctionMetadata import FunctionMetadata
-
+import pdb
+db = pdb.Pdb()
+db.prompt = '(coala-debugger)'
```

Comment:
I'm not sure if we should change the prompt. People should recognize pdb and look up it's documentation. If they only see 'coala-debugger' they moght expect extensive documentation from our side?

Reply:
Yes, you are right but change in color of string is needed :+1: